content/commands/ft.profile.md

```diff
@@ -102,14 +102,16 @@ If there's only one shard, the label will be omitted.
 | `Total`&nbsp;`profile`&nbsp;`time`     | The total run time (ms) of the query. Normally just a few ms. |
 | `Parsing`&nbsp;`time`           | The time (ms) spent parsing the query and its parameters into a query plan. Normally just a few ms. |
 | `Pipeline`&nbsp;`creation`&nbsp;`time` | The creation time (ms) of the execution plan, including iterators, result processors, and reducers creation. Normally just a few ms for `FT.SEARCH` queries, but expect a larger number for `FT.AGGREGATE` queries. |
+| `Total`&nbsp;`GIL`&nbsp;`time`     | The total time (ms) the query held the Global Interpreter Lock (GIL) during execution. Relevant for multi-threaded deployments where queries run in background threads. |
```

Comment:
@lerman25, Evntually, do we expect this field to exist in case of non multithreaded deployment? If so, what should be the value?

Reply:
No, if we don't have workers - this field doesn't print  (starting from https://github.com/RediSearch/RediSearch/pull/7756)